src/elements/quick-note/QuickNote.ts

```diff
@@ -394,6 +394,11 @@ export class QuickNoteElement extends OutsideClick implements OnInit, OnDestroy,
         let word = this.getWordAtCursor().trim();
         if (this.isTagging) {
             let symbol = this.config.triggers[this.taggingMode];
+            if (word.indexOf(symbol) === -1) {
+                this.zone.run(() => {
```

Comment:
So turns out I didn't need to run it in the angular zone and it was somehow just masking a bigger issue. Now it just returns empty string after it hides and works without running it in the angular zone